instrumentation/dalli/test/opentelemetry/instrumentation/dalli/instrumentation_test.rb

```diff
@@ -127,6 +127,14 @@
       _(span.name).must_equal 'set'
       _(span.attributes['db.statement']).must_equal 'set ?'
     end
+
+    it 'supports gat' do
+      dalli.gat('foo')
```

Comment:
`gat` was [added to `dalli` in 3.0.0](https://github.com/petergoldstein/dalli/blob/f72450aeb23a1a553299157f2a070eb421994b2c/CHANGELOG.md?plain=1#L178). We could `skip` if the dalli version is < 3.0.0 or set 3.0.0 as the min supported version.

Reply:
My preference is to `skip`, since we only need it in this test, and the test matrix _should_ cover 3.0.0.